modules/sov-ibc-transfer/src/rpc.rs

```diff
@@ -22,38 +17,20 @@ impl<S> IbcTransfer<S>
 where
     S: Spec,
 {
-    #[rpc_method(name = "escrowedToken")]
```

Comment:
now that we have ~bidirectional~ two maps - does it make sense to offer query methods for both of them?

Reply:
Couldn't think of any potential usage for that. But nice to have.

Reply:
[5c9bdaf](https://github.com/informalsystems/sovereign-ibc/pull/133/commits/5c9bdafd68d250b1cdd0183a3ee2708f3e03f6d5)